packages/components/src/Dropzone/sub-components/OnDragMessage/OnDragMessage.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react"
+import { Paragraph } from "@kaizen/typography"
+
+export const OnDragMessage = (): JSX.Element => (
+  <div className="flex flex-col items-center justify-center">
+    <span className="mb-8">
```

Comment:
You can add this class to `Paragraph` via `classNameOverride`

Reply:
If I do this the tailwind styles are the paragraph is inheriting the `Paragraph.module.scss` over the tailwind class.
<img width="605" alt="Screenshot 2023-03-23 at 4 42 49 pm" src="https://user-images.githubusercontent.com/43884313/227114348-b5b45459-6157-4fa9-bba9-ddc277d31a38.png">

To get around this, I need to use the `!important` flat. Is this something that you're okay with using in this component @gyfchong?